syntax.php

```diff
@@ -83,6 +83,9 @@ function handle($match, $state, $pos, Doku_Handler $handler){
         $urlparam = array();
         foreach($paramm as $key => $value) {
             switch($key) {
+                case 'list':
+                    $urlparam[] = $key . '=' . $paramm[$key];
```

Comment:
also ``$param[$key]`` is the same as ``$value``, so why not use that - it's a bit cleaner.